pkg/resources/helpers.go

```diff
@@ -61,3 +61,41 @@ func setBoolProperty(d *schema.ResourceData, key string, property *sdk.BoolPrope
 	}
 	return nil
 }
+
+func getTagObjectIdentifier(v map[string]any) sdk.ObjectIdentifier {
+	if _, ok := v["database"]; ok {
+		if _, ok := v["schema"]; ok {
+			return sdk.NewSchemaObjectIdentifier(v["database"].(string), v["schema"].(string), v["name"].(string))
+		}
+		return sdk.NewDatabaseObjectIdentifier(v["database"].(string), v["name"].(string))
+	}
+	return sdk.NewAccountObjectIdentifier(v["name"].(string))
+}
+
+func getPropertyTags(d *schema.ResourceData, key string) []sdk.TagAssociation {
```

Comment:
Was it not handled yet in any other resource?

Reply:
Not yet, as far as I checked everywhere getTags (resources/tag.go - last function) is used which is used for old SDK tags and this function is for new SDK tags